### PR TITLE
Create reset password page

### PR DESCRIPTION
**Pull request by whaught**
## Proposed Changes

* User can enter email and get a password-reset email sent

**Release Note**

```release-note
Include a password-reset page
```

### CODE REVIEW

File: cmd/server/assets/login/login.html [hunk 1]
```diff
@@ -35,14 +35,15 @@
...
+                <a class="card-link" href="login/resetpassword">Forgot password</a>
               </div>
             </div>
           </div>
         </div>
       </div>
     </div>
 
-    <div id="recaptcha-container" class="center-block"></div>
+    <div id=" recaptcha-container" class="center-block"></div>
```
Comment (sethvargo) [inline cmd/server/assets/login/login.html:46]: I dont think we want a space here?

File: cmd/server/assets/login/resetpassword.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+        let email = $email.val();
+
+        // Disable the submit button so we only attempt once.
+        $submit.prop('disabled', true);
+
+        firebase.auth().sendPasswordResetEmail(email).then(function() {
+          clearExistingFlash();
+          flash("Password reset email sent.", "success");
+          window.setTimeout(function() {
```
Comment (sethvargo) [inline cmd/server/assets/login/resetpassword.html:64]: Let's not redirect - we dont want them to miss the flash message, and there's a convenient back button on the page already

Comment (whaught) [reply cmd/server/assets/login/resetpassword.html]: I was debating this to myself.... removed

File: cmd/server/assets/login/login.html (hunk 1 above)
Comment (whaught) [reply cmd/server/assets/login/login.html]: I'ma blame my macbook keyboard somehow

**Pull request merged**